### ISSUE TITLE
What needs to be done to commit a new testnet ?

### ISSUE THREAD

**Issue by sbnair** [polkadot-js/apps#5245]
Hi
   Please let me know the steps to add a new testnet as it is running now in localhost but i would like to add this testnet into PolkaJS but i am unable to commit as i get commit is not allowed error when i try to push changes.  Please let me know what are the conditions to commit.

Thanks.

**Comment by jacogr** [comment]
1. For the repo on GH
2. Make the required changes to https://github.com/polkadot-js/apps/tree/master/packages/apps-config on your fork 
3. Make a PR from the fork

**Comment by sbnair** [comment]
Ok, thanks. I have done the steps that you mentioned and i have committed the changes into the forked repository, so what's next, should i need to see in workflow actions in this repo ?

**Comment by sbnair** [comment]
I had done this already (Pull request was created and closed) but i don't see anything in actions, is there anything that needs to be done ?

**Comment by jacogr** [comment]
External first-time contributors cannot run actions just by opening a PR - that is a hole GH closed recently due to wide-spread abuse. 

So all first-time contributors need an explicit approval from the maintainers (a button on the UI), before allowing GH actions to run on any new contributor PR.

See https://github.blog/changelog/2021-04-22-github-actions-maintainers-must-approve-first-time-contributor-workflow-runs/

So basically, for first-time contributors I click a button to allow CI to run - after reading through the PR.
